code/app/src/main/res/layout/mood_event_list_item.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Any specific reason as to why is this linear layout? I would prefer it to be constraint layout